bids/reports/tests/test_parameters.py

```diff
@@ -0,0 +1,302 @@
+import pytest
+import json
+import nibabel as nib
+
+from os.path import abspath, join
+
+from bids.layout import BIDSLayout
+from bids.reports import parameters
+from bids.tests import get_test_data_path
+
+
+@pytest.fixture
+def testlayout():
+    """A BIDSLayout for testing."""
+    data_dir = join(get_test_data_path(), "synthetic")
+    return BIDSLayout(data_dir)
+
+
+@pytest.fixture
+def testimg(testlayout):
+
+    func_files = testlayout.get(
+        subject="01",
+        session="01",
+        task="nback",
+        run="01",
+        extension=[".nii.gz"],
+    )
+    return nib.load(func_files[0].path)
+
+
+@pytest.fixture
+def testdiffimg(testlayout):
+
+    dwi_files = testlayout.get(
+        subject="01",
+        session="01",
+        datatype="dwi",
+        extension=[".nii.gz"],
+    )
+    return nib.load(dwi_files[0].path)
+
+
+@pytest.fixture
+def testconfig():
+    config_file = abspath(
+        join(get_test_data_path(), "../../reports/config/converters.json")
+    )
+    with open(config_file, "r") as fobj:
+        config = json.load(fobj)
+    return config
+
+
+@pytest.fixture
+def testmeta():
+    return {
+        "RepetitionTime": 2.0,
+        "MultibandAccelerationFactor": 2,
+        "ParallelReductionFactorInPlane": 2,
+        "FlipAngle": 90,
+        "PhaseEncodingDirection": "i",
+        "SliceTiming": [0, 1, 2, 3],
+    }
+
+
+@pytest.fixture
+def testmeta_light():
+    return {"RepetitionTime": 2.0}
+
+
+@pytest.mark.parametrize(
```

Comment:
This double parametrization leads to a lot of tests but they run fast, so I don't think this is a big issue except that it highlights that  `describe_sequence` does too many things.

Reply:
I agree that a lot of very fast tests (48 I think?) isn't a problem. I'm not sure how to improve `describe_sequence` (not because it's great, but because it's pretty simple), but happy to do so.

Reply:
Let's not worry about that, to be honest. We got bigger fish to fry. :fish: